src/main/java/taxi/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,26 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = (Long) req.getSession().getAttribute("driver_id");
+        List<Car> allCars = carService.getAllByDriver(driverId);
+        req.setAttribute("cars", allCars);
+        req.getRequestDispatcher("/WEB-INF/views/cars/all.jsp").forward(req, resp);
```

Comment:
i think our end point name shoud contain driver cars, not just cars,
something like this: /driver/cars
so those cars belong to current driver, not cars in general

Reply:
sorry for answering so late, but endpoint is 'drivers/cars', i just use the same jsp page ( "/WEB-INF/views/cars/all.jsp") for showing all cars for driver as for endpoint '/cars'.